### ISSUE TITLE
Cooling and Heating Simultaneous Operation in VSHeatPumpWaterToAirWithRHControl.idf Causes Severe Error

### ISSUE THREAD

**Issue by mitchute** [NREL/EnergyPlus#6323]
From issue #6316 

Heating loads don't converge after 40 days due to the VS coils operating at the same time when dehumidification is needed.

### Details
Some additional details for this issue (if relevant):
 - Platform: Windows 7 SP2
 - Version of EnergyPlus: Pre 8.8. 0c736af1ca11d92f9d639de67236f3c6051eecfb

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@mitchute @rraustad @Myoldmopar This file is still a problem, caused a timeout error in the integration coverage tests on #7163. I've added a priority label in hopes this gets some attention soon.

**Comment by mjwitte** [comment]
Adding a defect file here. The problem can be seen in the csv output, row 76,  01/03  03:57:30, the pertinent columns are all >0 which should never happen:
```
AY: SYS 1   HEAT PUMP COOLING MODE:Cooling Coil Total Cooling Rate [W](Each Call) = 24916
DP: SYS 1   HEAT PUMP HEATING MODE:Heating Coil Heating Rate [W](Each Call) = 23806
EA: HEAT PUMP DX SUPP HEATING COIL 1:Heating Coil Gas Rate [W](Each Call) = 390
```
[VSHeatPumpWaterToAirWithRHControl-OnlyRunPeriod1.idf.txt](https://github.com/NREL/EnergyPlus/files/2993255/VSHeatPumpWaterToAirWithRHControl-OnlyRunPeriod1.idf.txt)
Run with Miami TMY3.

**Comment by mjwitte** [comment]
After the fix (to be pushed shortly) the heat pump heating mode is always zero, cooling mode runs to dehumidify , and supp heat offsets that plus adds requested heating load. No more downstepping either on 01/03.

**Comment by Myoldmopar** [comment]
Closed via #7242
